fiscalyear.py

```diff
@@ -623,6 +623,10 @@ def fiscal_year(self):
         elif self in FiscalYear(self.year - 1):
             return self.year - 1
 
+    @property
+    def fiscal_month(self):
```

Comment:
Needs a docstring. Just copy the docstring from a similar method with the `:returns:` and `:rtype:` tags.

Reply:
Done